api/src/routers/meals_week1.js

```diff
@@ -0,0 +1,73 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const router_meal = express.Router();
+
+router_meal.get("/", async (req, res) => {
+  res.json({ Message: "Welcome to meal sharing App" });
+});
+
+router_meal.get("/future-meals", async (req, res) => {
+  try {
+    const meals = await knex.raw(
+      "SELECT * FROM meal WHERE `when` > NOW() ORDER BY `when` ASC"
+    );
+    res.json(meals[0]);
+  } catch (error) {
+    console.error(error);
+    res.status(404).json({ error: "Request not completed" });
```

Comment:
This is not the right error code. Try to look up HTTP error codes ([for example here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)) and make sure to use the right one, also for the other similar places in your code.

You don't need to get too specific - you can just send 500 (internal server error) in the catch blocks like this.

Reply:
got it, and changed accordingly.